tests/misc/image.txt

```diff
@@ -7,6 +7,6 @@
 
 ![Blank]()
 
-![Fail](http://humane man.jpg "The most humane man.")
+![Image](http://humane man.jpg "The most humane man.")
 
-![Fail](http://humane man.jpg)
+![Image](http://humane man.jpg)
```

Comment:
Rather than editing these, lets delete them and recreate the tests in the new syntax tests.